refinery/tool_manager/models.py

```diff
@@ -11,29 +12,37 @@
 from django.http import JsonResponse
 
 import bioblend
-from bioblend.galaxy.dataset_collections import (CollectionDescription,
-                                                 CollectionElement,
-                                                 HistoryDatasetElement)
+from bioblend.galaxy.dataset_collections import (
+    CollectionDescription, CollectionElement, HistoryDatasetElement
+)
```

Comment:
Did you find the setting to tweak in isort?

Reply:
Yeah! In an earlier PR: [see multi_line_output](https://github.com/refinery-platform/refinery-platform/blob/develop/.isort.cfg#L27)